mmv1/products/tpuv2/product.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: TpuV2
+display_name: Cloud TPU v2
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://tpu.googleapis.com/v2/
```

Comment:
should this be v2apha1 ?

Reply:
Done

Reply:
In an offline conversation with @rileykarson, the Terraform team would prefer to have the v2 version in the google-beta provider and then we can override with v2alpha1 when we implement the feature.